### ISSUE TITLE
As anon I can visit cashless.social and learn what it is and how it works so i can consider joining

### ISSUE THREAD

**Issue by utunga** [CashlessSociety/cashless-ui#26]

**Comment by DAHuerta** [comment]
@utunga Next steps. We would require a design for the landing page, and we also have to decide what content is going to be presented.

**Comment by utunga** [comment]
Yup this is another big one, a milestone all in its own, where really its more of a design and conceptualization challenge than a programming issue at least for the next bit.
